js/controller.js

```diff
@@ -857,7 +868,11 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         // If the core tells us that it will autoplay then we just display the loading
         // spinner, otherwise we need to render the big play button.
         if (params.willAutoplay) {
-          this.state.screenToShow = CONSTANTS.SCREEN.LOADING_SCREEN;
+          if (params.initialTime === 0) {
```

Comment:
Why is this screen shown only when `initialTime` is 0?

Reply:
\+